man/systemd-nspawn.xml

```diff
@@ -183,11 +185,13 @@
         <listitem><para>If specified, the container is run with a temporary snapshot of its file system that is removed
         immediately when the container terminates. May not be specified together with
         <option>--template=</option>.</para>
-        <para>Note that this switch leaves host name, machine ID and all other settings that could identify the
-        instance unmodified. Please note that — as with <option>--template=</option> — taking the temporary snapshot is
-        more efficient on file systems that support subvolume snapshots or 'reflinks' naively (<literal>btrfs</literal>
-        or new <literal>xfs</literal>) than on more traditional file systems that do not
-        (<literal>ext4</literal>).</para>
+        <para>Note that this switch leaves host name, machine ID and all other settings that could identify
+        the instance unmodified. Please note that — as with <option>--template=</option> — taking the
+        temporary snapshot is more efficient on file systems that support subvolume snapshots or 'reflinks'
+        naively (<literal>btrfs</literal> or new <literal>xfs</literal>) than on more traditional file
```

Comment:
natively!

Reply:
Ouch and @gdamjan even had pointed this out already over in #11054 but I forgot to fix that... (Sorry!)